core/src/avm2/globals/flash/display/graphics.rs

```diff
@@ -365,6 +367,264 @@ fn draw_rect<'gc>(
     Ok(Value::Undefined)
 }
 
+lazy_static! {
+    /// Length between two points on a unit circle that are 45 degrees apart from
+    /// one another.
+    ///
+    /// This constant is `H`, short for 'hypotenuse', because it is also the length
+    /// of the hypotenuse formed from the control point triangle of any quadratic
+    /// Bezier curve approximating a 45-degree unit circle arc.
+    ///
+    /// The derivation of this constant - or a similar constant for any other arc
+    /// angle hypotenuse - is as follows:
+    ///
+    /// 1. Call the arc angle `alpha`. In this special case, `alpha` is 45 degrees,
+    ///    or one-quarter `PI`.
+    /// 2. Consider the triangle formed by the center of the circle and the two
+    ///    points at the start and end of the arc. The two other angles will be
+    ///    equal, and it and `alpha` sum to 180 degrees. We'll call this angle
+    ///    `beta`, and it is equal to `alpha` minus 180 degrees, divided by 2.
+    /// 3. Using the law of sines, we know that the sine of `alpha` divided by `H`
+    ///    is equal to the sine of `beta` divided by `r`, where `r` is the radius
+    ///    of the circle. We can solve for `H` to get the result. Note that since
+    ///    this is a unit circle, you won't see a radius term in this constant.
+    static ref H: f64 = (PI * 0.25).sin() / (PI * 0.375).sin();
```

Comment:
Alternatively, you can just write the formula but hardcode the result as `const` - this way we avoid the `lazy_static` for simple float constants.
Like here:
https://github.com/ruffle-rs/h263-rs/blob/master/h263/src/decoder/cpu/idct.rs#L36

(again, up to your preference)